taskcluster/ci/startup-test/kind.yml

```diff
@@ -30,6 +30,29 @@ task-defaults:
     run-on-tasks-for: []
 
 tasks:
+    fenix-arm-nightly-robo-opt:
+        attributes:
+            build-type: fenix-nightly
+            shipping-product: fenix
+        description: Fenix Nightly Opt Robo Test (ARM)
+        dependencies:
+            signing: signing-apk-fenix-nightly
```

Comment:
Aaron and I have been talking about this offline: I'm not sure if this `signing-apk-fenix-nightly` task is something that is generated during runtime when a `signing-apk` task for the build type `fenix-nightly` is executed.

If so, then should be fine to depend on.

Reply:
I'll double check this week here (been swamped).

Reply:
Thanks for following up, this is great to hear!